index.html

```diff
@@ -66,10 +66,11 @@ <h4 class="content__mission">We know how large objects will act, but things on a
 
     <section class="strong-points">
         <div class="container">
+
             <div class="strong-points__content">
                 <h2 class="content__title">Why choose us</h2>
                 <p class="content__text">Problems trying to resolve the conflict between <br> the two major realms of
```

Comment:
"br" not the best option to use, it's better to set styles in css, just assign a max-width and align to the center )

Reply:
I'll keep this in mind for the future, thank you for your advice!